install-ci.yml

```diff
@@ -77,6 +77,17 @@
     - role: dd-nodepool
       tags:
         - monitoring
+    - role: logrotate
+      logrotate_configs:
+        - name: nodepool
+          path: /var/log/nodepool/*log
+          options:
+            - compress
+            - missingok
+            - rotate 10
```

Comment:
What's the effect if we have the python lib rotation + the logrotate rotation on at the same time? We should figure that out before applying this.